cli/command/formatter/stats.go

```diff
@@ -145,8 +145,10 @@ func (c *containerStatsContext) Container() string {
 
 func (c *containerStatsContext) Name() string {
 	c.AddHeader(nameHeader)
-	name := c.s.Name[1:]
-	return name
+	if len(c.s.Name) > 1 {
+		return c.s.Name[1:]
+	}
+	return ""
```

Comment:
This change is only for avoiding panic, in my opinion, we should fix this on daemon side, we should pass the name of container to client in all situations. Then `c.s.Name` will never be empty.

To me, "--" or "" are the same, they both represent the wrong cases.